infrastructure/ansible/roles/traffic_ops/tasks/traffic_ops.yml

```diff
@@ -134,10 +140,19 @@
     - production/log4perl.conf
     - production/riak.conf
     - cdn.conf
-    - ldap.conf
     - influxdb.conf
   notify: Restart Traffic Ops
 
+- name: Render Traffic Ops LDAP configuration file
+  template:
+    src: "ldap.conf.j2"
+    owner: "{{ to_user }}"
+    group: "{{ to_group }}"
+    mode: 0600
+    dest: "{{ to_conf_installdir }}/ldap.conf"
+  notify: Restart Traffic Ops
+  when: to_ldap_setup == "yes"
```

Comment:
This needs to use actual boolean evaluation instead of string matching

Reply:
Done